src/inputs/TextField/textField.stories.tsx

```diff
@@ -50,8 +51,40 @@ export const readOnly = () => {
         label="Name"
         readOnly
         value={value}
-        onChange={e => setValue(e.target.value)}
+        onChange={(e) => setValue(e.target.value)}
```

Comment:
why is typescript compiling and not showing error there because of untyped `e`?

Reply:
because it can infer it:
![image](https://user-images.githubusercontent.com/4376509/86018083-3b3ced80-b9fb-11ea-8bee-578b3d12ea63.png)

The `onChange` it's already typed.

Reply:
But this is about what type of `e` you will receive, not what you expect. I'd prefer explicit typing, so in case we expect a different type of `e`, an error will be shown